openmc/mgxs/mgxs.py

```diff
@@ -1479,7 +1505,10 @@ def get_pandas_dataframe(self, groups='all', nuclides='all',
                 distribcell_paths=distribcell_paths)
 
         # Remove the score column since it is homogeneous and redundant
-        df = df.drop('score', axis=1)
+        if self.domain_type == 'mesh':
+            df = df.drop('score', axis=1, level=0)
```

Comment:
Oh I see. I wish Pandas had a slimmer API with a single family of index types, but oh well.